src/realm/object-store/sync/app.cpp

```diff
@@ -989,7 +989,14 @@ void App::call_function(std::shared_ptr<SyncUser> user, const std::string& name,
         if (auto error = AppUtils::check_for_errors(response)) {
             return completion_block(error, util::none);
         }
-        completion_block(util::none, util::Optional<bson::Bson>(bson::parse(response.body)));
+        bson::Bson body_as_bson;
```

Comment:
Declaring this variable as util::Optional<bson::Bson> is slightly simpler than converting to an optional later.